### PR TITLE
[JENKINS-65605] Bridge methods are no longer present

### PR DESCRIPTION
**Pull request by jtnord**
See [JENKINS-65605](https://issues.jenkins-ci.org/browse/JENKINS-65605).

### Proposed changelog entries

* Restore bridge methods used to retain backwards compatibility. (regression in 2.278)

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [x] (If applicable) Jira issue is well described
- [x] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [x] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

**Comment by jtnord** [comment]
just waiting for a failed build before pushing the fix..

**Comment by jtnord** [comment]
> just waiting for a failed build before pushing the fix..

well it worked locally from the IDE, and did not run in Jenkins because I named the test wrong.....  so rather than wait even longer, just fixeing the test name and including the fix :-/

File: test/src/test/java/test/compatability/BridgeMethodsTest.java
Comment (basil) [inline test/src/test/java/test/compatability/BridgeMethodsTest.java:22]: Extra [

**Review by basil — APPROVED** [review]

File: test/src/test/java/test/compatability/BridgeMethodsTest.java
Comment (basil) [inline test/src/test/java/test/compatability/BridgeMethodsTest.java:1]: s/compatability/compatibility

File: test/src/test/java/test/compatability/BridgeMethodsTest.java
Comment (MarkEWaite) [inline test/src/test/java/test/compatability/BridgeMethodsTest.java:22]:
```suggestion
    @Issue("JENKINS-65605")
```

Comment (jtnord) [reply test/src/test/java/test/compatability/BridgeMethodsTest.java]: fixed in b36eec37598edfd20dc063e5380421befe1c03f0

File: test/src/test/java/test/compatability/BridgeMethodsTest.java
Comment (jtnord) [reply test/src/test/java/test/compatability/BridgeMethodsTest.java]: fixed in b36eec37598edfd20dc063e5380421befe1c03f0

**Review by timja — APPROVED** [review]

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

---

FYI @res0nance

**Review by jglick — APPROVED** [review]

File: test/src/test/java/test/compatibility/BridgeMethodsTest.java
Comment (jglick) [inline test/src/test/java/test/compatibility/BridgeMethodsTest.java:1]: Does not match https://github.com/jenkinsci/jenkins/blame/b36eec37598edfd20dc063e5380421befe1c03f0/test/src/test/java/test/README.md#L1 BTW.

Comment (jtnord) [reply test/src/test/java/test/compatibility/BridgeMethodsTest.java:1]: where would you suggest that tests go that do not check specific functionality?  
I expliclty did not want to put this in `hudson.model.QueueTest` or indeed in any `hudson.` `jenkins.` as it is not scoped to that level (even if it is using the Queue for testing purposes)

Comment (jglick) [reply test/src/test/java/test/compatibility/BridgeMethodsTest.java:1]: Right…`jenkins` package? Or `jenkins.bugs`? No strong opinion.

Comment (jtnord) [reply test/src/test/java/test/compatibility/BridgeMethodsTest.java]: addressed in fc967a6ba40ebf23e134f27b4c651facf3fe6e9f

**Review by jglick — APPROVED** [review]

**Review by basil — APPROVED** [review]

**Review by res0nance — APPROVED** [review]

**Review by daniel-beck — APPROVED** [review]

**Pull request merged**
